app/client/css/styles.scss

```diff
@@ -94,6 +94,16 @@ body {
     }
 }
 
+.skychat-halo-preview {
+    border:1px solid white;
+    width:14px;
+    height:14px;
+    border-radius:50%;
+    background:transparent;
+    display:inline-block;
+    margin-right:4px;
+}
+
```

Comment:
Please add a space between the property and values
e.g. `border: 1px solid white;`

Reply:
done